### PR TITLE
format is treated as simple string; Removed 'Format' type

### PR DESCRIPTION
**Pull request by navinrathore**
FORMAT_XLSX removed it being redundant and not addable as pointing to same string as XLS.

### CODE REVIEW

File: client/src/main/java/zingg/client/pipe/PipeFactory.java [hunk 1]
```diff
@@ -25,18 +25,17 @@ public static void register(String name, Pipe p) {
 	private static final Pipe getPipe(Pipe p) {
 		try {
 			switch (p.format) {
-			case CSV:
-			case JSON:
-			case XLS:
-			case XLSX:
+			case Pipe.FORMAT_CSV:
+			case Pipe.FORMAT_JSON:
```
Comment (sonalgoyal) [inline client/src/main/java/zingg/client/pipe/PipeFactory.java:29]: why do we need these cases? 

also what is the use of jdbc pipe?

File: python/zingg/pipes/pipes.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import logging
-from zingg import *
+from python.zingg import *
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:2]: the namespace is zingg

File: python/zingg/pipes/pipes.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
-Format = jvm.zingg.client.pipe.Format
+JPipe = jvm.zingg.client.pipe.Pipe
 FilePipe = jvm.zingg.client.pipe.FilePipe
 
 class CsvPipe(Pipe):
     def __init__(self, name):
-        Pipe.__init__(self, name, Format.CSV.type())
+        Pipe.__init__(self, name, JPipe.FORMAT_CSV)
         Pipe.addProperty(self, FilePipe.HEADER,"true")
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:13]: why true? I made the same comment earlier too?

File: python/zingg/pipes/pipes.py [hunk 4]
```diff
@@ -20,7 +20,7 @@ def setLocation(self, location):
 
 class BigQueryPipe(Pipe):
     def __init__(self,name):
-        Pipe.__init__(self, name, "bigquery")
+        Pipe.__init__(self, name, JPipe.FORMAT_BIGQUERY)
         Pipe.addProperty(self, "viewsEnabled", "true")
```
Comment (sonalgoyal) [inline python/zingg/pipes/pipes.py:24]: ?

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: python/zingg/pipes/pipes.py (hunk 3 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:13]: It was automerge issue

File: python/zingg/pipes/pipes.py (hunk 4 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:24]: It was automerge issue

File: python/zingg/pipes/pipes.py (hunk 2 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:2]: It was erroneously came. Though, I have to add it every time to go through. Not sure there is any environment setting to work with python stuff.

File: client/src/main/java/zingg/client/pipe/PipeFactory.java (hunk 1 above)
Comment (sonalgoyal) [reply client/src/main/java/zingg/client/pipe/PipeFactory.java:29]: there is no need to have these cases if they logically do not make sense now. plz rm

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

File: python/zingg/pipes/pipes.py (hunk 3 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:13]: The comments was not for this PR. It was for below PR 
https://github.com/zinggAI/zingg/pull/374#discussion_r913319871
and has been taken care there.

File: python/zingg/pipes/pipes.py (hunk 4 above)
Comment (navinrathore) [reply python/zingg/pipes/pipes.py:24]: The comments was not for this PR. It was for below PR
https://github.com/zinggAI/zingg/pull/374#discussion_r913319871
and has been taken care there.

File: client/src/main/java/zingg/client/pipe/Pipe.java [hunk 5]
```diff
@@ -61,12 +74,12 @@ public void setName(String name) {
...
-	public Format getFormat() {
+	public String getFormat() {
 		return format;
 	}
 	
 	@JsonValue
-	public void setFormat(Format sinkType) {
+	public void setFormat(String sinkType) {
 		this.format = sinkType;
 		PipeFactory.register(name, this);
```
[the remark this answers is not in the document]
Comment (navinrathore) [reply client/src/main/java/zingg/client/pipe/Pipe.java:84]: No. shall remove.

File: client/src/test/java/zingg/client/TestArguments.java [hunk 6]
```diff
@@ -265,13 +264,13 @@ public void testWriteArgumentObjectToJSONFile() {
 
 				Pipe inputPipe = new Pipe();
 				inputPipe.setName("test");
-				inputPipe.setFormat(Format.CSV);
+				inputPipe.setFormat(Pipe.FORMAT_CSV);
 				inputPipe.setProp("location", "examples/febrl/test.csv");
```
[the remark this answers is not in the document]
Comment (navinrathore) [reply client/src/test/java/zingg/client/TestArguments.java:268]: Not sure. need to check.

File: core/src/main/java/zingg/util/PipeUtil.java [hunk 7]
```diff
@@ -312,44 +311,44 @@ public static void writePerSource(Dataset<Row> toWrite, Arguments args, JavaSpar
...
 	public static Pipe getBlockingTreePipe(Arguments args) {
 		Pipe p = new Pipe();
-		p.setFormat(Format.PARQUET);
+		p.setFormat(Pipe.FORMAT_PARQUET);
 		p.setProp(FilePipe.LOCATION, args.getBlockFile());
 		p.setMode(SaveMode.Overwrite);
 		return p;
 	}
 
 	public static String getPipesAsString(Pipe[] pipes) {
```
[the remark this answers is not in the document]
Comment (navinrathore) [reply core/src/main/java/zingg/util/PipeUtil.java:349]: Removed.

**Pull request closed without merging**